aws-event-bridge/src/test/scala/akka/stream/alpakka/aws/eventbridge/EventBridgePublishMockSpec.scala

```diff
@@ -0,0 +1,176 @@
+
+    probe
+      .sendNext(requestDetail("eb-message-1"))
+      .sendNext(requestDetail("eb-message-2"))
+      .sendNext(requestDetail("eb-message-3"))
+      .sendComplete()
+
+    Await.result(future, 1.second) mustBe putResult :: putResult :: putResult :: Nil
+
+    val expectedFirst = requestDetail("eb-message-1")
+    verify(eventBridgeClient, times(1)).putEvents(meq(expectedFirst))
+
+    val expectedSecond = requestDetail("eb-message-2")
+    verify(eventBridgeClient, times(1)).putEvents(meq(expectedSecond))
+
+    val expectedThird = requestDetail("eb-message-3")
+    verify(eventBridgeClient, times(1)).putEvents(meq(expectedThird))
+  }
+
+  it should "publish multiple PublishRequest messages to multiple eventbridge topics" in {
+    val putResult = resultResponse("event-id")
+
+    when(eventBridgeClient.putEvents(any[PutEventsRequest]())).thenReturn(CompletableFuture.completedFuture(putResult))
+
+    val (probe, future) =
+      TestSource.probe[PutEventsRequest].via(EventBridgePublisher.publishFlow()).toMat(Sink.seq)(Keep.both).run()
+
+    probe
+      .sendNext(requestDetail("eb-message-1"))
+      .sendNext(requestDetail("eb-message-2"))
+      .sendNext(requestDetail("eb-message-3"))
+      .sendComplete()
+
+    Await.result(future, 1.second) mustBe putResult :: putResult :: putResult :: Nil
+
+    val expectedFirst = requestDetail("eb-message-1")
+    verify(eventBridgeClient, times(1)).putEvents(meq(expectedFirst))
+
+    val expectedSecond = requestDetail("eb-message-2")
+    verify(eventBridgeClient, times(1)).putEvents(meq(expectedSecond))
+
+    val expectedThird = requestDetail("eb-message-3")
+    verify(eventBridgeClient, times(1)).putEvents(meq(expectedThird))
+  }
+
+  it should "publish a single PutEventsRequestEntry message to eventbridge" in {
+    val putRequestEntry = PutEventsRequestEntry.builder().detail("event-bridge-message").build()
+    val publishRequest = requestDetail("event-bridge-message")
+    val putResult = resultResponse("event-id")
+
+    when(eventBridgeClient.putEvents(meq(publishRequest))).thenReturn(CompletableFuture.completedFuture(putResult))
+
+    val (probe, future) =
+      TestSource.probe[PutEventsRequestEntry].via(EventBridgePublisher.flow()).toMat(Sink.seq)(Keep.both).run()
+    probe.sendNext(putRequestEntry).sendComplete()
+
+    Await.result(future, 1.second) mustBe putResult :: Nil
+    verify(eventBridgeClient, times(1)).putEvents(meq(publishRequest))
+  }
+
+  it should "publish multiple PutEventsRequestEntry messages to eventbridge" in {
+    val putResult = resultResponse("event-id")
+
+    when(eventBridgeClient.putEvents(any[PutEventsRequest]())).thenReturn(CompletableFuture.completedFuture(putResult))
+
+    val (probe, future) =
+      TestSource.probe[Seq[PutEventsRequestEntry]].via(EventBridgePublisher.flowSeq()).toMat(Sink.seq)(Keep.both).run()
+    probe
+      .sendNext(Seq(entryDetail("eb-message-1")))
```

Comment:
This, or another, test should probably test sending multiple messages in a `Seq` as a single record.